torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1498,27 +1500,39 @@ void InitXlaModuleBindings(py::module m) {
             if (!backend_data) {
               continue;
             }
-            auto* infoptr =
-                static_cast<torch::lazy::LazyGraphExecutor::DeviceDataInfo*>(
-                    backend_data->info());
-            XLA_CHECK(infoptr);
 
             // Dedup by handle
             xla::ComputationClient::Data::OpaqueHandle handle =
                 backend_data->GetHandle();
             if (!data_handles.insert(handle).second) {
               continue;
             }
-
-            // add tensor_id after we make sure the handle does not exist yet.
-            tensor_ids.push_back(infoptr->tensor_id);
+            auto* infoptr =
+                static_cast<torch::lazy::LazyGraphExecutor::DeviceDataInfo*>(
+                    backend_data->info());
+            if (infoptr) {
+              tensor_ids.push_back(infoptr->tensor_id);
+            } else {
+              // TODO(JackCaoG): Make sure this device data is actually seed.
```

Comment:
yep, I'm also worried if there would be DeviceData node other than seed that has null InfoPtr

Reply:
I will add the check in a following pr.